scanner/src/main/kotlin/scanners/ScanCode.kt

```diff
@@ -81,7 +81,11 @@ import okio.sink
  *   be considered in [getConfiguration]. Only used if the [log] level is set to [Level.DEBUG]. Defaults to
  *   [DEFAULT_DEBUG_NON_CONFIGURATION_OPTIONS].
  */
-class ScanCode(name: String, config: ScannerConfiguration) : LocalScanner(name, config) {
+class ScanCode(
+    name: String,
+    config: ScannerConfiguration,
+    val licenseFileMatcher: LicenseFileMatcher = LicenseFileMatcher.DEFAULT_MATCHER
```

Comment:
This should be `private`, or at least `internal`.

Reply:
done.